pkg/ocm/versions.go

```diff
@@ -248,9 +248,12 @@ func IsValidVersion(userRequestedVersion string, supportedVersion string, cluste
 	if err != nil {
 		return false, err
 	}
+
+	isPreRelease := a.Prerelease() != "" && b.Prerelease() != ""
+
 	versionSplit := a.Segments64()
 	//If user has specified patch we check directly and return the result
-	if len(versionSplit) > 2 && versionSplit[2] > 0 {
+	if len(versionSplit) > 2 && versionSplit[2] > 0 || isPreRelease {
```

Comment:
If the user has specified patch or is a pre release version we can compare directly